src/CraigslistAPI/CraigslistPostDetailsComponent.js

```diff
@@ -25,7 +26,15 @@ export default class CraigslistPostDetailsComponent extends React.Component {
                 <h2 className="mt-3">{this.state.listing.title}</h2>
                 <h4>{this.state.listing.price}</h4>
                 <div className="row">
-                    <div className="col"><p>{this.state.listing.description}</p></div>
+                    <div className="col">
+                        <p>{this.state.listing.description}</p>
+                        <br/>
+                        <p>Posted: {parseDate(this.state.listing.date)}</p>
```

Comment:
Ooh, and something worth discussion is having the the readable, parsed date appear in the post details like we have currently, but in the list/grid views upon the initial search, having something like "Posted n days ago" instead of the exact date. What do you think?

Reply:
Sounds like a good idea that we might want to implement. Let's discuss this at a later time and experiment with it. For now, the date is a good first iteration.